modules/age.nix

```
@@ -53,7 +53,7 @@ let
         type = types.str;
         default = config._module.args.name;
         description = ''
-          Name of the file used in /run/agenix
+          Name of the file used in ''${cfg.secretsDir}
```

Do we really want to escape this? If you include this documentation shows up in `man configuration.nix`, it'll show up as `${cfg.secretsDir}` instead of its value (e.g. the default `/run/agenix`), right?

(I can check this myself if you're not sure)

```suggestion
          Name of the file used in ${cfg.secretsDir}
```

---

I want to indicate that the value is dynamic. If you change `secretsDir` then the description is inaccurate because its not the name of the file in `/run/agenix`. I believe this method is used in some NixOS modules also.

---

Maybe I'm misunderstanding, but by interpolating the value of the variable, it will print exactly the path that it is set to. If you escape it like `''${cfg.secretsDir}`, it will show exactly `${cfg.secretsDir}` in the documentation. If you don't escape it and have `age.secretsDir = "whatever the value of secretsDir is";`, it should show `whatever the value of secretsDir is`.

---

Ah, gotcha. Yeah, I think that's fine then.